packages/dso-toolkit/src/styles/components/_modal.scss

```diff
@@ -169,6 +169,10 @@ body.dso-modal-open {
           margin-top: $u1;
         }
 
+        &.btn:not(.dso-list-button) + .btn:not(.dso-list-button) {
```

Comment:
In plaats van dit nu TEGEN te gaan... heb je ook overwogen om misschien in `button_base()` juist die regel:
```
  &:not(.dso-list-button) + .btn:not(.dso-list-button) {
    margin-left: $u2;
  }
```
... alleen te laten gelden voor de grotere viewports?